config/routes.rb

```diff
@@ -11,7 +11,11 @@
       get :music_index
       patch :sort
     end
+
   end
 
+  get 'reviews', to: 'reviews#index'
+  post 'reviews', to: 'reviews#create'
```

Comment:
You may want to use named routes for your get and post routes if these are going to non-resourceful.
For example:
get 'reviews', to: 'reviews#index', as: recent_reviews